opendbc/safety/safety/safety_toyota.h

```diff
@@ -206,6 +208,7 @@ static bool toyota_tx_hook(const CANPacket_t *to_send) {
       desired_accel = to_signed(desired_accel, 16);
 
       bool violation = false;
+      violation |= toyota_secoc && (desired_accel != TOYOTA_LONG_LIMITS.inactive_accel);  // SecOC cars move this signal to 0x183
       violation |= longitudinal_accel_checks(desired_accel, TOYOTA_LONG_LIMITS);
```

Comment:
this should be written more clearly

Reply:
@adeebshihadeh are you asking for the logic to be rewritten or make the comment more clear? Can you give a little more insight into what you would like to see here?

Reply:
I tried making the logic and comment more clear. lmk if you want additional changes

Reply:
Updated to remove bitwise